alchemy-service/src/main/java/io/rtr/alchemy/service/AlchemyService.java

```diff
@@ -42,7 +42,7 @@ public void initialize(final Bootstrap<T> bootstrap) {
 
     @Override
     public void run(final T configuration, final Environment environment) throws Exception {
-        Preconditions.checkState(configuration instanceof AlchemyServiceConfiguration);
+        Preconditions.checkArgument(configuration instanceof AlchemyServiceConfiguration);
```

Comment:
this one i think is an actual state exception, no?  it should only fail when starting the application

Reply:
i think that sounds right